spec/unit/provider/package/zypper_spec.rb

```diff
@@ -248,6 +248,13 @@ def shell_out_expectation!(*command, **options)
         )
         provider.remove_package(["emacs"], ["1.0"])
       end
+
+      it "should run zypper remove for non existing package" do
+        shell_out_expectation!(
+          "zypper", "--non-interactive", "remove", "noname=1.0"
+        )
```

Comment:
this expectation doesn't have anything to do with the changed code so its difficult to see how this is relevant and looks like its not really testing the behavior.

Reply:
Maybe if the description is changed, "should run zypper remove for non-existent package without raising an error"?

Reply:
Hi @marcparadise Thanks for the suggestion. I've updated the spec description.